app/models/users/setting.rb

```diff
@@ -2,7 +2,8 @@ module Users
   class Setting < ApplicationRecord
     self.table_name_prefix = "users_"
 
-    belongs_to :user
+    belongs_to :user, touch: true
```

Comment:
Is this being used for cache busting?

Reply:
(I had to think about your question for a few minutes - cos I was like "what is cache busting?" 🤔 )

Yes! yes we are. When certain settings (eg `config_theme`) are changed, in order for a refresh to apply the changes, the cached user [here](https://github.com/forem/forem/blob/8d632d14478a3ca14fb09d4b8fd6f567e03945cf/app/controllers/async_info_controller.rb#L76) needs to be updated.

I hope that makes sense; I'm explaining it as best I understand.

Reply:
That makes perfect sense. Knowing that we're caching content for user settings makes this make a lot of sense. Thanks for the link to where it's used, too! 🎉